Overview.src.html

```diff
@@ -2036,6 +2040,18 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
   <span title=concept-network-error>network error</span> and
   <var>response</var> is not a
   <span title=concept-filtered-response>filtered response</span>, set
+  <var>response</var>'s
+  <span title=concept-response-cors-exposed-headers-list>CORS-exposed headers list</span> to the result of
+  <span title=concept-header-parse>parsing</span>
+  `<code title=http-access-control-expose-headers>Access-Control-Expose-Headers</code>` in
+  <var>response</var>'s
+  <span title=concept-response-header-list>header list</span>.
+
+ <li>
+  <p>If <var>response</var> is not a
+  <span title=concept-network-error>network error</span> and
+  <var>response</var> is not a
+  <span title=concept-filtered-response>filtered response</span>, set
```

Comment:
For now I've just separated out the check for network error/filtered response, and then have substeps to set the CORS-exposed header-name list and create the filtered response. I agree that somehow making creating a CORS filtered response more of a single step would be nice, but I'm not sure how best to do so...